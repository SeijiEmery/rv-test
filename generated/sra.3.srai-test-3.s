srai x1, x1, 1
srai x2, x2, 1
srai x3, x3, 1
srai x4, x4, 1
srai x5, x5, 1
srai x6, x6, 1
srai x7, x7, 1
srai x8, x8, 1
srai x9, x9, 1
srai x10, x10, 1
nop
nop
nop
nop
nop
ebreak
