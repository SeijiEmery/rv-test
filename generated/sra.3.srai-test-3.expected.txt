tests/sra.test.s:33
R1 = 0
R2 = 1
R3 = 1
R4 = 2
R5 = 2
R6 = 3
R7 = 3
R8 = 4
R9 = 4
R10 = 5
